index.html

```diff
@@ -218,7 +218,7 @@ <h2>Tonic</h2>
       </section>
 
       <section id="about">
-        <div>
+          <div class="desktop-about textside">
           <h2>About<br />Myself</h2>
           <p>
             Hello I’m a software developer! I can help you build a product ,
```

Comment:
- [ ] This section consists of two parts - `About Myself` part and `Language` part. The about myself part takes up 40% of the width while the language part takes up the remaining 60% according to how we have it on the Figma design. Please, take a look at both designs, compare and work your design alongside the Figma design.
- [ ] Also, consider adjusting the vertical margin among `lets connect`,  `social icons`,  and  `Get my resume` button.

### Your Design
![your-about-desktop](https://user-images.githubusercontent.com/26597432/220648355-78acaf15-7d70-4937-8ecc-4245180f99c3.png)

### Figam Design
![temp-about-desktop](https://user-images.githubusercontent.com/26597432/220648406-5e03fe62-6868-44fb-a443-26d569fc05b4.png)

Reply:
Done